payments/credit-card/index.md

```diff
@@ -56,7 +56,7 @@ receive the state of the transaction.
 You will redirect the payer to Swedbank Pay hosted pages to collect the credit
 card information.
 
-![Screnshot-1]
+![Card-payment][card-payment]{:height="500px" width="425px"}
```

Comment:
The content inside the first set of brackets should be a human readable text that will be read out loud by screen readers for people with low or no eyesight. `Card-payment` is a bit sparse as a description, no?